src/installer/tests/helixpublish.proj

```diff
@@ -45,9 +46,16 @@
     <MSBuild Projects="%(HostTestProject.Identity)" Targets="GetTargetPath">
       <Output TaskParameter="TargetOutputs" PropertyName="_TargetPath" />
     </MSBuild>
+
+    <PropertyGroup>
+      <_TestExePath>$([System.IO.Path]::GetFileNameWithoutExtension($(_TargetPath)))$(ExeSuffix)</_TestExePath>
+    </PropertyGroup>
+
     <ItemGroup>
       <HostTestProject>
-        <Command>dotnet test $([System.IO.Path]::GetFileName($(_TargetPath))) $(TestRunnerAdditionalArguments) --logger trx --results-directory .</Command>
+        <Command Condition="'$(TargetOS)' == 'windows'">$(_TestExePath) $(TestRunnerAdditionalArguments) --report-trx --results-directory .</Command>
```

Comment:
Best to double check if this produces the TRX in the right place. I'm honestly not very sure if it does or not.
Also, I'm not sure if the TRX file name could be different than what it originally used to be. So in case you rely on file name patterns when publishing in CI etc, it's worth double checking that it's getting published correctly.